lib/msf/core/module.rb

```diff
@@ -278,6 +278,14 @@ def fail_with(reason, msg=nil)
     raise RuntimeError, "#{reason.to_s}: #{msg}"
   end
 
+  # Whether {user_data} contains everything necessary to make a
+  # `MetasploitDataModels::AutomaticExploitation::MatchResult`
+  #
+  # @return [bool]
+  def user_data_is_match?
```

Comment:
Personally, I like to add _obj in these cases, like user_data_match_obj or user_data_is_match_obj